src/contracts/IdleRouter.sol

```diff
@@ -56,18 +64,40 @@ contract IdleRouter is OwnableUpgradeable {
         _deposit(_token, _amount, true);
     }
 
+    /**
+     * @notice deposit tokens into the AA CDO Tranche. Note: approvals must be made before
+     * this call to allow the router to move your assets on your behalf
+     * @param _token the underlying token for the CDO
+     * @param _amount of the underlying token to deposit
+     */
     function depositBB(address _token, uint256 _amount) external {
         _deposit(_token, _amount, false);
     }
 
+    /**
+     * @notice burn AA Tranche tokens and get the principal + interest back
```

Comment:
Might want to add a note here also that users will need to grant the router approval to move its _tranceToken as well.